### ISSUE TITLE
[Coupon] - Accessibility - Incorrect VoiceOver for the coupon announcement button

### ISSUE THREAD

**Issue by jostnes** [woocommerce/woocommerce-ios#6605]
**Describe the bug**
One of the VoiceOver on the coupon page is incorrect, it reads some random number instead of what the button is. 

**To Reproduce**
1. From Settings > Experimental Features > Turn on Coupon Management
2. Go to Hub Menu > Coupons
3. With VoiceOver turned on, go to the expanded button on the announcement
4. VoiceOver reads a long number instead of describing what the button is

**Screenshots**
The highlighted button:
![IMG_0039](https://user-images.githubusercontent.com/17252150/162416934-dc9d68db-8ffb-4daa-9d92-2c3cf71f0733.PNG)

**Expected behavior**
VoiceOver should describe the button correctly

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: iPhone 13
- iOS version: 15.3.1
- WooCommerce iOS version: Version 8.9 (pr6586-e96dae9)

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by ThomazFB** [comment]
@jostnes looks like this issue got fixed already, I was comparing how the voiceOver was behaving around multiple banners and the announcement looks correct to me now, the selection for the expand/collapse button now includes the Banner title in the announcement text. Could you confirm if everything is working as expected on your end too? Thanks!

Screenshots
==========
| Before  | After |
| ------------- | ------------- |
| ![IMG_9086](https://user-images.githubusercontent.com/5920403/168707436-76e303ba-bf14-4603-b355-35924db22543.PNG) | ![IMG_9085](https://user-images.githubusercontent.com/5920403/168707423-aa417cc1-da69-4c40-b53d-b9a4f3c0d357.PNG) |

**Comment by jostnes** [comment]
Yup looks like this is fixed now (I tested on latest beta WCiOS 9.2)! I no longer hear the random number being read on VoiceOver and as you mentioned the expand/collapse button seems to be included in the Banner title as well. I'll close this since it's fixed. 

Thanks for the ping and for checking this issue out @ThomazFB!
